### ISSUE TITLE
Error compiling on EL6

### ISSUE THREAD

**Issue by durval-menezes** [gordan-bobic/zfs-fuse#13]
Hello Gordan, 

With the latest version available here (Latest commit 5f140ce), scons aborts with the following error:

```
cc1: error: unrecognized command line option "-fstrict-volatile-bitfields"
scons: *** [cmd/mount_zfs/mount_zfs.o] Error 1
scons: building terminated because of errors.
```

Possibly gcc version? Mine is the EL6 standard:

```
gcc (GCC) 4.4.7 20120313 (Red Hat 4.4.7-16)
```

Cheers, 
   Durval.

**Comment by ElCoyote27** [comment]
Durval,
Just remove '-fstrict-volatile-bitfields' and it will work/compile fine.
if you're using the spec file to build rpms, you won't run into this issue as the spec file has this code:
%{__perl} -pi -e 's@-fstrict-volatile-bitfields@@' lib/libumem/Makefile.in SConstruct

( '-fstrict-volatile-bitfields' must be removed from both SConstruct and lib/libumem/Makefile.in)
Cheers,
Vincent

**Comment by durval-menezes** [comment]
Hello Vincent, 

Thanks for your response, and you are correct: I just ran the one-liner you mentioned, ie:

```
 perl -pi -e 's@-fstrict-volatile-bitfields@@' lib/libumem/Makefile.in SConstruct
```

just before running scons, and it was enough to enable it to run successfully. 

Perhaps this should be added to the INSTALL file; in fact, I just did it,  here's the (trivial) patch: 
[INSTALL.patch.txt](https://github.com/gordan-bobic/zfs-fuse/files/53131/INSTALL.patch.txt)

Anyway, I'm closing this issue.

Cheers, 
Durval.
